doc-Deployment_Planning_Guide/topics/planning.adoc

```diff
@@ -40,11 +40,29 @@ The Region also uses a region database that reports to a master database on the
 {product-title} can be configured in a highly available setup. In this case, all PostgreSQL instances must be running on a server that is deployed from the {product-title} appliance. 
 ifdef::cfme[]
 For more information, see the following example reference architecture:
-https://access.redhat.com/articles/1571263[Implementing Highly Available CloudForms Appliances]. Note, this reference architecture was written for a  previous version of CloudForms.
+https://access.redhat.com/articles/1571263[Implementing Highly Available CloudForms Appliances]. Note, this reference architecture was written for a previous version of CloudForms.
 endif::cfme[]
 ====
 
+[[central-administration]]
+=== Centralized Administration
 
+{product-title} includes centralized administration capabilities with support for the following life cycle management operations that can be initiated from the global region and processed and executed on the remote region.
+
+* Virtual machine provision from the global region
+* Invoke service provision request from the global region  
+* Virtual machine power operations from the global region
+* Virtual machine retirement from the global region
+* Virtual machine reconfigure from the global region
+
```

Comment:
Hi @carbonin, @bdunne: I've modified some of the operations referring to the pivotal trackers, could you review please?